src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -324,7 +324,7 @@ public ValidatedAddress ValidateAddress(string address)
             // P2WSH (unsupported)
             else if (BitcoinWitScriptAddress.IsValid(address, this.Network, out Exception _))
             {
-                result.IsValid = true;
+                result.IsValid = false;
```

Comment:
The correct fix is to probably remove this last two lines from Stratis network
```
            var encoder = new Bech32Encoder("bc");
            this.Bech32Encoders = new Bech32Encoder[2];
            this.Bech32Encoders[(int)Bech32Type.WITNESS_PUBKEY_ADDRESS] = encoder;
            this.Bech32Encoders[(int)Bech32Type.WITNESS_SCRIPT_ADDRESS] = encoder;
```

Reply:
Would removing that bech32 code make  BitcoinWitScriptAddress.IsValid() return false?

Reply:
looks like it yes, because it looks for those enums in the IsValid (didn't try it though)

Reply:
Interesting... Ok I'll give it a try and see if it works. Looks like I also have a conflict to resolve now as well so will do that too!